### DISCUSSION TITLE
Wrong model for billing

### DISCUSSION THREAD

**Discussion by wdxwj** [langfuse/langfuse#2098] [Support] [answered]
When I call OPENAI, I don't pass the model_name. It will be gpt3.5 by default. But you just take the gpt3.5, not the real one from Response from OPENAI. So the billing is wrong. I am using langchain.

**Comment by marcklingen** [comment, marked as answer]
Can you create a bug report with some code to reproduce this issue? We are happy to have a look and fix this

**Reply by wdxwj** [reply]
Sure
